index.html

```diff
@@ -5061,6 +5061,143 @@ <h2 id="AudioContextInfo">
             </dl>
           </section>
         </section>
+        <section>
+          <h2 id="instantiation-of-AudioWorkletNode-and-AudioWorkletProcessor">
```

Comment:
Will you remove the existing description of the AudioWorkletNode constructor? It seems redundant now.